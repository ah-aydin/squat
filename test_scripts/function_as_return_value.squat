func add(int number1, int number2) int {
    return number1 + number2;
}

func get_func() func(int, int) int {
    return add;
}

func main() {
    println(get_func()(10, 20));
}
