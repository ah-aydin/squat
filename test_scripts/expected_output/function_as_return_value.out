Compiling and running file: test_scripts/function_as_return_value.squat
30
